frontend/angular/README.md

```diff
@@ -0,0 +1,23 @@
+## Introduction
+Angular is a popular front-end framework made by Google. Like other popular front-end frameworks, it uses a component-based architecture to structure apps.
+
+If you are a beginner for Angular and wonder how to start, This article is for you and we focus only on basic Angular App with one page. Once finished our app looks like below where we can add items, edit, delete items.
+
+## How to get started with Angular
+
+-   Install [Nodejs](https://nodejs.org/en/download/)
+-   Install Angular CLI
+-   Install Typescript globally
+
+  Once you install nodejs and we need to install Angular cli and typescript globally with the following commands.
+  
+    npm install -g @angular/cli
+    npm install -g typescript 
+  
+## Goal
+-   Provide quick start template for projects with flavour of Bootstrap.
+-   Provide a range of features that are common to large classes of projects.
+
+## Pre-requisites
+-   Basic understanding of Typescript/Javascript
+-   Basic knowledge about Angular
```

Comment:
Can't add this in pre-requisite. The whole idea of the repo is that we provide them with some basic knowledge of any tech to start with along with a template and some links to advance tutorials.

Reply:
I have deleted the pre-requisites taking into what you had said above. Please review @harsh-2711